lang/da.js

```diff
@@ -110,7 +110,7 @@ export default {
 	"components.tag-list.cleared-item": "Fjernede taglisteelement {value}",
 	"components.tag-list.interactive-label": "Tagliste, {count} elementer",
 	"components.tag-list.num-hidden": "+ {count} mere",
-	"components.tag-list.role-description": "Tag List",
+	"components.tag-list.role-description": "Tagliste",
 	"components.tag-list.show-less": "Vis færre",
 	"components.tag-list.show-more-description": "Vælg for at få vist skjulte taglisteelementer",
 	"components.tag-list-item.role-description": "Tag",
```

Comment:
Looks like for a few languages here (CY, DA, DE) "tag" translates to "tag" in those languages as well. Just to avoid the validator complaining at us, can we add `// mfv-translated` to those lines?

```suggestion
	"components.tag-list-item.role-description": "Tag", // mfv-translated
```